src/utils/compileJsx.ts

```diff
@@ -1,19 +1,45 @@
 import { transform } from 'sucrase';
+import { parse } from '@babel/parser';
```

Comment:
Could we get away with [`parseExpression`](https://babeljs.io/docs/babel-parser#babelparserparseexpressioncode-options) instead? We are technically only parsing a single expressions right?

Reply:
> When in doubt, use `.parse()`

I was in doubt.

Reply:
Done https://github.com/seek-oss/playroom/pull/304/commits/286e42c2a9dd39b96a734a10031d75994e527736